components/eamxx/src/physics/p3/tests/CMakeLists.txt

```diff
@@ -103,42 +107,13 @@ if (SCREAM_ENABLE_BASELINE_TESTS)
       EXCLUDE_MAIN_CPP
       ${FORCE_RUN_DIFF_FAILS})
 
-  #
```

Comment:
By using a ctest + a test label (i.e., `ctest -L baseline` rather than `make baseline`), we don't really need to jump hoops to get the test launch line and args from the test.